### PR TITLE
Misc updated to private prefetch proxy post for clarity

### PR DESCRIPTION
**Pull request by tunetheweb**
Changes proposed in this pull request:

- Adds location and mime-type of PPP configuration file
- Adds info on how to disable
- Adds links

FYI @KenjiBaheux

### CODE REVIEW

**Review by malchata — APPROVED** [review]
LGTM, with the caveat that I'm not an expert on private prefetch proxy.

**Comment by tunetheweb** [comment]
Thanks @malchata , will wait on the API owners (who asked for this!) to review this before merging anyway.

File: site/en/blog/private-prefetch-proxy/index.md
Comment (buettner) [inline site/en/blog/private-prefetch-proxy/index.md:77]: "how much Private Prefetch proxy should let through" -> ".. how much *traffic the* Private Prefetch Proxy should let through."

"JSON to in an " -> "JSON to a ..."

"..served with `application..." -> ".. served with the..."

File: site/en/blog/private-prefetch-proxy/index.md
Comment (buettner) [inline site/en/blog/private-prefetch-proxy/index.md:86]: Up to you, but this sentence is a dupe from above and can probably be simplified by dropping the first sentence.

e.g., "The fraction field is a float between 0.0.."

File: site/en/blog/private-prefetch-proxy/index.md
Comment (buettner) [inline site/en/blog/private-prefetch-proxy/index.md:97]: "The /.well-known/traffic-advice file is fetched by the proxy, not the client, and cached at the proxy per the usual HTTP cache semantics. "

**Review by buettner — COMMENTED** [review]
A few comments but otherwise LGTM. 

Thanks!

File: site/en/blog/private-prefetch-proxy/index.md
Comment (tunetheweb) [reply site/en/blog/private-prefetch-proxy/index.md:77]:
```suggestion
From past experiments, we know that this feature typically results in less than 2% extra requests for main resources (for example HTML documents). That said, if you are the cautious kind, you can use the traffic advice's fraction field to control how much traffic the Private Prefetch Proxy should let through. You can start with a small fraction such as 0.3 (that is, 30%), and gradually increase it to 1.0 (that is, 100%) by adding the following JSON to a `/.well-known/traffic-advice` file, which needs to be served with the `application/trafficadvice+json` MIME type:
```

File: site/en/blog/private-prefetch-proxy/index.md
Comment (tunetheweb) [reply site/en/blog/private-prefetch-proxy/index.md:86]:
```suggestion
The `fraction` field is a float between 0.0 (no prefetch at all) and 1.0 (100% of the prefetch requests get through).
```

File: site/en/blog/private-prefetch-proxy/index.md
Comment (tunetheweb) [reply site/en/blog/private-prefetch-proxy/index.md:97]:
```suggestion
The `/.well-known/traffic-advice` file is fetched by the proxy, not the client, and cached at the proxy per the usual HTTP cache semantics. For more flexibility—for example, a sudden peak of heavy access—you may want to temporarily reject prefetch requests (`Sec-Purpose: prefetch;anonymous-client-ip`) with a 503 status code, and by setting the `Cache-Control: no-store` header on the response. You may also add the [`Retry-After`](https://tools.ietf.org/html/rfc7231#section-6.6.4) header to tell Chrome how long to wait before retrying prefetch requests.
```

**Comment by jeremyroman** [comment]
lgtm2

**Pull request merged**
